### ISSUE TITLE
[CAL-1182] Merge and reuse components settings/conferencing and installed/[category]

### ISSUE THREAD

**Issue by sean-brydon** [calcom/cal.com#7521]
Both of these pages have heavy overlap and a lot of repeated code across them - we can consolidate them into more reusable components to save having to make changes in two places (As conferencing category on installed does the exact same as settings :) ) 

@Jaibles Is having duplicate pages with the same results intended?

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-1182](https://linear.app/calcom/issue/CAL-1182/merge-and-reuse-components-settingsconferencing-and)</sub>

**Comment by lmnnarciso** [comment]
Can I be assigned on this issue? I'll look into it but might take a while doing it.

**Comment by sean-brydon** [comment]
Sure! @lmnnarciso assigned you

**Comment by lmnnarciso** [comment]
@sean-brydon Saw the the settings/conferencing page from `/settings/my-account/conferencing` but not sure of where the installed/[category] is located.

**Comment by Jaibles** [comment]
> @Jaibles Is having duplicate pages with the same results intended?

Yes @sean-brydon this is intended. Calendars & conferencing are so important for people to have setup that it is deemed essential setting. We had many queries in v1 of Cal where people didn't find it in Installed Apps. However, it wouldn't make sense to remove it from Installed Apps just because we have it in settings, since it is technically an installed app.

In both instances we should use the list item components and of course re-use code where possible like you suggested
[List item component](https://www.figma.com/file/9MOufQNLtdkpnDucmNX10R/%E2%9D%96-Cal-DS?node-id=27844%3A90422&t=i2Fyksx6LtxaUtCG-1)

**Comment by PeerRich** [comment]
@lmnnarciso any update?

**Comment by raju249** [comment]
@PeerRich - Can you as assign it to me. I can work on this and prefer to mess up someone else's time by 2 PRs.

PS: I have experience contributing to OSS, I come from GitLab 😄

**Comment by raju249** [comment]
Ok, so I had an initial look at both the files, since I am new to TypeScript and in process of learning it, I found below blocks, very similar in both the files.

In `conferencing.tsx` - https://github.com/calcom/cal.com/blob/main/apps/web/pages/settings/my-account/conferencing.tsx#L135-L154

And in `installed/[category]` - https://github.com/calcom/cal.com/blob/main/apps/web/pages/apps/installed/%5Bcategory%5D.tsx#L167-L187

Basically one of the drop down menu items is staggeringly similar, does this issue intend to merge the menu item and hook it at both the places?

If so, I would need some help on how do I do it. 😅
